dart/lib/src/sentry.dart

```diff
@@ -33,12 +35,14 @@ class Sentry {
     OptionsConfiguration optionsConfiguration, [
     AppRunner appRunner,
     List<Integration> initialIntegrations,
-  ]) async {
+    PlatformChecker platformChecker
```

Comment:
what's about adding a new method:

```
@VisibleForTesting
static Future<void> init(SentryOptions options, [appRunner, initialIntegrations]) {
}
```

so you move the things from the actual one to this new one, but this one is only not private because of testing, but ideally, it should not be used by the client.

Reply:
nvm about this last comment, there's no method overload.

the `init` method now takes an optional `SentryOptions`, so you could create your own SentryOptions instance and set a PlatformChecker mock before calling the init method.

You need to pull the repo :)